ddtrace/llmobs/_integrations/langgraph.py

```diff
@@ -3,7 +3,7 @@
 from typing import List
 from typing import Optional
 
-from ddtrace import tracer
+from ddtrace._trace.span import Span
```

Comment:
Any reason we can't just do `from ddtrace.trace import Span` like the other changes?

Reply:
I wanted to avoid circular imports in different ddtrace products. So I blindled updated all deprecated references to import from `ddtrace._trace.span`. I will clean this up. Ideally different products should not rely on tracing internals.